docs/index.rst

```diff
@@ -110,8 +65,20 @@ Indices and tables
 ------------------
 
 * :ref:`genindex`
-* :ref:`modindex`
 * :ref:`search`
 
-Documentation last updated on |today|
+.. note::
```

Comment:
I was trying to make top of the documentation less visually 'loud' by having fewer of the styled callout boxes, but I see your point here. I've moved it back up in the latest commit.